api/app/Models/PoolCandidate.php

```diff
@@ -411,4 +411,35 @@ public function isDraft()
     {
         return is_null($this->submitted_at) || $this->submitted_at->isFuture();
     }
+
+    /**
+     * Scope the query to PoolCandidate's the current user can view
+     */
+    public function scopeauthorizedToView(Builder $query)
```

Comment:
Casing again - scopeAuthorizedToView

Reply:
Fixed in [fix scope case](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6064/commits/49c451143e61f09be2caa988e15c3aa140db8893)